### PR TITLE
Recommendation for formatting style in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by JuliaKukulies**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (JuliaKukulies) [reply CONTRIBUTING.md:39]: In the current PR, the code is only checked for compliance and not formatted directly. I agree, that an auto-formatting of pull requests would be practical, but a caveat is that such action would change the code in the same commit, I think. The history is then much more difficult to comprehend,since the made changes are not highlighted anymore (as you can see in my formatting PR). Maybe, it is possible to run an auto-formatting and change the PR in a new commit, but I guess than it could be problematic with pull requests from private forks....I can have a look at that again.

Comment (w-k-jones) [reply CONTRIBUTING.md:39]: I had a look into this, unfortunately you can't commit to a PR using github actions as it's a security issue. There is the possibility that we could set up a bot account to do this with maintainer status, but I'm not sure what that would involve and it's probably quite a bit of work for not a huge benefit!

Comment (JuliaKukulies) [reply CONTRIBUTING.md:39]: Thank you for looking into this @w-k-jones! My personal opinion would be that it is OK to require that contributors do the formatting themselves and run black on the files they have changed.

**Comment by w-k-jones** [comment]
I'm happy to merge this now, but we should also make sure that the contributing.md file is added to the main and dev branches

**Review by w-k-jones — APPROVED** [review]

**Comment by fsenf** [comment]
Hi @JuliaKukulies , @w-k-jones  , @freemansw1 ,

this PR is the next in the row for our formatting topic. Two ideas:

1. directly after PR merge, we need to get the contributing files to the other branch `main` & `dev`
2. I would prefer to have the bugfix in the spectral filtering in a spectral PR

Would do you think? Cheers, Fabian.

**Pull request merged**
